helpdesk_mgmt_stage_validation/models/validate_utils.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _
+from odoo.exceptions import ValidationError
+
+
+def validate_stage_fields(records):
+    for rec in records:
+        stage = rec.stage_id
+        field_ids = stage.validate_field_ids
+        field_names = [x.name for x in field_ids]
+        values = rec.read(field_names)
+        for name in field_names:
+            if not values[0][name]:
+                raise ValidationError(
+                    _(
+                        'Cannot move to stage "%s" '
+                        'until the "%s" field is set.' % (stage.name, name)
+                    )
```

Comment:
Don't do string replacement into translate function, even if the translations can't be generated correctly.

```suggestion
                    _(
                        'Cannot move to stage "%s" '
                        'until the "%s" field is set.'
                    )  % (stage.name, name)
```

And please add the ticket name into the error message.

And if you can give all the missing fields on all tickets in 1 message it can be cool.
Even if, if you have by example 5 missing fields on 3 tickets, you will have 15 exceptions one by one after fixing each one by one.